recipes/tutorials/basic_project_setup.ipynb

```diff
@@ -0,0 +1,415 @@
+{
```

Comment:
<p>We should probably remove the actual organization ID from here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kili-technology/kili-python-sdk/pull/896/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>